spec/puppet-lint/plugins/check_classes/arrow_on_right_operand_line_spec.rb

```diff
@@ -11,7 +11,8 @@
 
       context 'arrow on the line of left operand' do
         let(:code) do
-          "Package['httpd']  #{operator}
+          "
+            Package['httpd']  #{operator}
```

Comment:
First of all, I'm _very_ sorry this slipped by me for so long. That said, I don't have any strong opinions on the alignment. It could be something we want to add to Style Guide, but I'd likely be asking @rnelson0 , David Schmitt, Henrik, and/or Hunter for opinions on it.